packages/governance/src/binaryBallotCounter.js

```diff
@@ -1,12 +1,19 @@
 // @ts-check
 
 import { assert, details as X } from '@agoric/assert';
-import { makeStore } from '@agoric/store';
-import { makePromiseKit } from '@agoric/promise-kit';
+import { E } from '@agoric/eventual-send';
 import { Far } from '@agoric/marshal';
+import { makePromiseKit } from '@agoric/promise-kit';
+import { sameStructure } from '@agoric/same-structure';
+import { makeStore } from '@agoric/store';
 
-import { E } from '@agoric/eventual-send';
-import { ChoiceMethod, buildBallot } from './ballotBuilder.js';
+import {
+  ChoiceMethod,
+  buildBallot,
+  looksLikeQuestionForType,
+  positionIncluded,
+  looksLikeBallotSpec,
+} from './ballotBuilder.js';
 import { scheduleClose } from './closingRule.js';
 
 const makeWeightedBallot = (ballot, shares) => harden({ ballot, shares });
```

Comment:
What does this make and return? From the name, I figured it was some kind of WeightedBallot. But I couldn't find anything matching in types.js, or even anything with a property named `shares`.